WordPress/src/main/java/org/wordpress/android/ui/compose/components/buttons/ButtonSize.kt

```diff
@@ -0,0 +1,10 @@
+package org.wordpress.android.ui.compose.components.buttons
+
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.dp
+
+enum class ButtonSize(val height: Dp?) {
+    NORMAL(height = null),
```

Comment:
💡 Would it make sense to have an actual value for `NORMAL` instead of `null` so height doesn't have to be nullable?

Reply:
Good point, I was taking a look now and it seems the default that is considered "empty" (or `Wrap Content` maybe) is `Dp.Unspecified`, I will change the code to use that.